internal/validate/helpers.go

```diff
@@ -79,3 +90,85 @@ func ReadFile(ctx context.Context, fileName string) (string, error) {
 	log.Debugf("Loaded %s", fileName)
 	return string(fileBytes), nil
 }
+
+// PreProcessPolicy fetches policy sources and returns a policy object with
+// pinned SHA/image digest URL where applicable, along with a policy cache object.
+func PreProcessPolicy(ctx context.Context, policyOptions policy.Options) (policy.Policy, *cache.PolicyCache, error) {
```

Comment:
This should probably go under `internal/policy`?

Reply:
Moved.